### DISCUSSION TITLE
Hardness

### DISCUSSION THREAD

**Discussion by NecromancyStudios** [SkriptLang/Skript#5267] [Suggestions] [answered]
Is there a way to change the hardness of a block?

**Comment by NecromancyStudios** [comment]
nvm there is

**Reply by NotSoDelayed** [reply]
If you have any questions related to Skript, kindly head to https://discord.gg/skript and ask there instead.

**Comment by abandonedaccount6235** [comment, marked as answer]
Discussions aren't intended for getting help writing scripts. Please check out the "relevant links" in the repo readme for places you can get help.
